src/otk/traversal.py

```diff
@@ -1,3 +1,20 @@
 class State:
-    def __init__(self, path):
+    def __init__(self, path, defines):
         self.path = path
+        self.defines = defines
+
+    def copy(self, *, path=None, defines=None) -> "State":
```

Comment:
(super nitpick) the commit message talks about `new()` but this is now `copy()`

Reply:
oh no, I missed that!

Reply:
It seems it still does(?) (c.f. https://github.com/osbuild/otk/pull/126/commits/d1ab6046f376079c2862016a6b8f954920720b70)

Reply:
Fixed.